### PR TITLE
ci/scripts: travis cleanup

### PR DESCRIPTION
**Pull request by Beerosagos**
Travis is not used anymore: this deletes unused related files and refactor scripts to remove travis refs.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -35,9 +35,9 @@ jobs:
       - name: Run CI script
         # The script also runs golang-ci but it's ok: doesn't take too long and may be useful
         # to keep its linter errors in this log, too.
-        run: ./scripts/travis-ci.sh ci
+        run: ./scripts/github-ci.sh ci
         env:
-          TRAVIS_OS_NAME: linux
+          GITHUB_OS_NAME: linux
```
Comment (benma) [inline .github/workflows/ci.yml:40]: TRAVIS_OS_NAME is a real env var in travis. GITHUB_OS_NAME is not a real env var in GH CI. Maybe best to rename it to simply OS_NAME or CI_OS_NAME.

**Review by benma — CHANGES_REQUESTED** [review]
Typo in commit: trevis -> travis

**Comment by Beerosagos** [comment]
Updated, thanks! PTAL

**Review by benma — COMMENTED** [review]
LGTM

**Pull request merged**